github-actions/unified-status-check/src/labels.ts

```diff
@@ -0,0 +1,47 @@
+import {actionLabels, targetLabels} from '../../../ng-dev/pr/common/labels.js';
+import {PullRequest} from './pull-request.js';
+import {ValidationFunction} from './validation.js';
+
+export const checkForTargelLabel: ValidationFunction = (pullRequest: PullRequest) => {
+  const appliedLabel = Object.values(targetLabels).find((label) =>
+    pullRequest.labels.includes(label.name),
+  );
+  if (appliedLabel !== undefined) {
+    return {
+      state: 'success',
+      description: `Pull request has target label: "${appliedLabel.name}"`,
+    };
+  }
+  return {
+    state: 'pending',
+    description: 'Waiting for target label on the pull request',
+  };
+};
+
+export const isMergeReady: ValidationFunction = (pullRequest: PullRequest) => {
+  if (!pullRequest.labels.includes(actionLabels.ACTION_MERGE.name)) {
+    return {
+      state: 'pending',
+      description: `Waiting for "${actionLabels.ACTION_MERGE.name}" label on the pull request`,
+    };
+  }
+
+  if (pullRequest.labels.includes(actionLabels.ACTION_REVIEW.name)) {
+    return {
+      state: 'failure',
+      description: `Pull request is marked for merge but still has the "${actionLabels.ACTION_REVIEW.name}" label on the pull request`,
+    };
+  }
+
+  if (pullRequest.labels.includes(actionLabels.ACTION_CLEANUP.name)) {
+    return {
+      state: 'failure',
+      description: `Pull request is marked for merge but still has the "${actionLabels.ACTION_CLEANUP.name}" label on the pull request`,
+    };
+  }
+
+  return {
+    state: 'success',
+    description: `Pull request is marked for merge by the "${actionLabels.ACTION_MERGE.name}" label on the pull request`,
```

Comment:
I believe these messages are way too long to display. Can you try shorten these?